### PR TITLE
[do-not-merge] apply the updated movefmt tool to framework code

### PR DESCRIPTION
**Pull request by robinlzw**
## Description
This PR shows the result after applying the newest [movefmt v1.0.3](https://github.com/movebit/movefmt/tree/develop) to the framework code using the default option:
```
max_width = 90
indent_size = 4
hard_tabs = false
tab_spaces = 4
emit_mode = "NewFiles"
verbose = "Normal"
prefer_one_line_for_short_branch_blk = true
```

### CODE REVIEW

File: aptos-move/framework/aptos-framework/sources/account.move [hunk 1]
```diff
@@ -233,7 +246,7 @@ module aptos_framework::account {
                 key_rotation_events,
                 rotation_capability_offer: CapabilityOffer { for: option::none() },
                 signer_capability_offer: CapabilityOffer { for: option::none() },
-            }
+            },
```
Comment (rahxephon89) [inline aptos-move/framework/aptos-framework/sources/account.move:249]: @robinlzw, do you remember there are comments about this added comma?

File: aptos-move/framework/aptos-framework/sources/aggregator_v2/aggregator_v2.move [hunk 2]
```diff
@@ -67,17 +67,22 @@ module aptos_framework::aggregator_v2 {
...
         aggregator.max_value
     }
 
     /// Creates new aggregator, with given 'max_value'.
     ///
     /// Currently supported types for IntElement are u64 and u128.
     /// EAGGREGATOR_ELEMENT_TYPE_NOT_SUPPORTED raised if called with a different type.
-    public native fun create_aggregator<IntElement: copy + drop>(max_value: IntElement): Aggregator<IntElement>;
+    public native fun create_aggregator<IntElement: copy + drop>(max_value: IntElement): Aggregator<
+        IntElement>;
```
Comment (rahxephon89) [inline aptos-move/framework/aptos-framework/sources/aggregator_v2/aggregator_v2.move:81]: This is a bug?

File: aptos-move/framework/aptos-framework/sources/account.move (hunk 1 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/account.move:249]: Of course, I'll fix this today first, it's a simple issue.

File: aptos-move/framework/aptos-framework/sources/aggregator_v2/aggregator_v2.move (hunk 2 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/aggregator_v2/aggregator_v2.move:81]: yeah, we created issue about it in https://github.com/movebit/movefmt/issues/16

Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/aggregator_v2/aggregator_v2.move:81]: fixed, you can see at https://github.com/movebit/movefmt/issues/16

**Comment by vineethk** [comment]
@robinlzw Could you redo this with the latest formatter?

**Pull request closed without merging**